shellbase-core/src/main/scala/com/sumologic/shellbase/ShellCommand.scala

```diff
@@ -46,7 +46,9 @@ abstract class ShellCommand(val name: String,
 
   protected def currentCommand: String = _currentCommand.get()
 
-  def executeLine(arguments: List[String], commandPath: List[String] = List()): Boolean = {
+  def executeLine(argumentsInput: List[String], commandPath: List[String] = List()): Boolean = {
```

Comment:
_technically_ renaming is a breaking change, because you could do `executeLine(arguments = blah)`.  I'm not sure how strict we should be on that though